packages/plexus/src/DirectedGraph/builtins/EdgePath.tsx

```diff
@@ -15,6 +15,8 @@
 import * as React from 'react';
 
 type TProps = Record<string, any> & {
+  id: string;
```

Comment:
do we need to change anything under deprecated `DirectedGraph/..`? Can your PR work without that?

Reply:
Yes, that was for when I tried `DirectedGraph`, similar changes have been made to `Diagraph`, so we can skip these